test/e2e/framework/helpers.go

```diff
@@ -18,3 +22,41 @@ func (f *Framework) MustCreateOrUpdateConfigMap(t *testing.T, cm *v1.ConfigMap)
 		t.Fatalf("failed to create/update configmap - %s", err.Error())
 	}
 }
+
+// MustGetConfigMap or fail
```

Comment:
I would maybe improve the godoc a bit with something like:
```suggestion
// MustGetConfigMap tries to get the `namespace/name` ConfigMap by polling it for 5 minutes. 
// It fails if the ConfigMap is not found within the polling period.
```